fun main() {
    // Write some code to cause an exception
    print(5 /0)
}